src/main/java/org/example/TaskResource.java

```diff
@@ -12,36 +12,36 @@
 /** Example resource class hosted at the URI path "/tasks"
  */
 @Path("/tasks")
-public class TaskResource {
+public class TaskResource implements ITodoDAO {
 
     @Path("/create")
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
     public ImmutableTask createNewTask(ImmutableTask newTask) {
-        TodoService.createTask(newTask);
-        return newTask;
+
+        return createTaskDB(newTask);
     }
 
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     public ArrayList<ImmutableTask> getAllTasks() {
-        return TodoService.getAllTasks();
+        return getAllTasksDB();
     }
 
     @Path("/update")
     @PUT
     @Consumes(MediaType.APPLICATION_JSON)
     public Response update(ImmutableTask task) {
-        TodoService.updateTask(task);
+        updateTaskDB(task);
```

Comment:
@dagibu301 
The reason i'm asking is because you're catching the exception in the DAO Implementation, while i'm not familiar with Java would seems like no matter what happens the response will always be successful right ?